.tekton/run_unittests.sh

```diff
@@ -50,6 +50,9 @@ echo    "with dependencies in '${BUNDLE_GEMFILE}'"
 gem update --system > /dev/null
 echo "Gem version $(gem --version)"
 
+# Verify if net-http is included in the installation (no separate gem needed)
```

Comment:
I find this comment a bit misleading. The gem is always included, no need to verify that.
And it is always a "separate" gem, just like the rest.
The question is whether the included one is the latest or an older one.

Reply:
Modified the comment to [this](https://github.com/instana/ruby-sensor/pull/388/files#diff-7634e6a9f9a1d293732ac9239731ed4fbbafbf1ad312b79b4d5486f7ce2369f8R53).

> The question is whether the included one is the latest or an older one.
It is the latest version (0.4.1)